akeyless_gateway/README.md

```diff
@@ -0,0 +1,104 @@
+
+## Setup
+
+Akeyless offers a unique Gateway, which adds an extra level of protection between your private network and the cloud. Acting as a SaaS extension of our core services, our stateless Gateway enables a transparent internal operation with a robust out-of-the-box mechanism to ensure service continuity and recovery while you are not required to change any network infrastructure in order to work with your internal resources.
+
+To configure the integration with Datadog to view important Akeyless Gateway metrics, follow the below instructions for the method you are using (or have used) for your Gateway deployment.
+
+### Prerequisites
+- An Akeyless Gateway either running or being deployed for the first time
+
+### Configure
+
+This integration works with one Gateway or multiple instances using the same API key. Metrics can be shown per `host` or `instance` in the **Akeyless GW** dashboard.
+
+#### For a Gateway Running on Kubernetes
+
+To configure the Akeyless Gateway integration on a [Gateway running on K8s](https://docs.akeyless.io/docs/gateway-k8s):
+
+1. In your `values.yaml` file you use to deploy your Gateway on Kubernetes, under the `metrics` section, add the below configuration. Set the relevant API Key of your Datadog server, and the relevant site. If your Datadog server is running in the EU site, use `datadoghq.eu`. Default is `datadoghq.us`.
+
+```
+metrics:
+  enabled: true  
+  config: |
+    exporters:    
+      datadog:
+        api:
+          key: "<Your Datadog API key>"
+          site: <Your Datadog server site>         
+    service:
+      pipelines:
+        metrics:
+          exporters: [datadog]
+```
+
+2. If you have not yet deployed the Gateway, continue with your Installation as usual and run the following command when ready to deploy:
+
+```
+helm install <your-gateway-name> akeyless/akeyless-api-gateway -f values.yaml
+```
+
+3. If you are updating an existing Gateway on Kubernetes, run the following commands to update:
+
+```
+helm upgrade <your-gateway-name> akeyless/akeyless-api-gateway -f values.yaml
+```
+
+#### For a Standalone Gateway Running on Docker
+
+To configure the Akeyless Gateway integration on a [Standalone Gateway](https://docs.akeyless.io/docs/install-and-configure-the-gateway):
+
+1. Create a local file called `otel-config.yaml` with the below configuration. Set the relevant API Key of your Datadog server, and the relevant site. If your Datadog server is running in the EU site, use `datadoghq.eu`. Default is `datadoghq.us`.
+
+```
+exporters:
+  datadog:
+    api:
+      key: "<Your Datadog API key>"
+      site: <Your Datadog server site>
+service:
+  pipelines:
+    metrics:
+      exporters: [datadog]
+```
+
+2. If you have not yet deployed the Gateway, run the below command to spin up your Akeyless Gateway with the `ENABLE_METRICS=true` variable and mounting the `otel-config.yaml` file.
```

Comment:
```suggestion
2. If you have not yet deployed the Gateway, run the below command to spin up your Akeyless Gateway with the `ENABLE_METRICS=true`, mounting the `otel-config.yaml` file:
```

Reply:
Let's change it as follows:

"If you have not yet deployed the Gateway, run the below command to spin up your Akeyless Gateway with metrics enabled and the config file mounted:"

Reply:
Done. Made the change on my end.